e2e/mocks/data-api/person-query.json

```
@@ -19,7 +19,7 @@
       "PriorUWNetIDs": [],
       "PriorUWRegIDs": [],
       "RegisteredFirstMiddleName": "Jane Doe",
```

I believe RegisteredFirstMiddleName should match RegisteredName 
```suggestion
      "RegisteredFirstMiddleName": "Janeth Doe",
```